src/main.py

```diff
@@ -58,6 +58,9 @@ def get_player_cookie(log: Logger, driver: uc_chrome) -> str:
 
     chrome_options = uc_chrome_options()
     chrome_options.add_argument("--headless")
+    chrome_options.add_argument("--disable-gpu")
+    chrome_options.add_argument("--no-sandbox")
+    chrome_options.add_argument("--disable-dev-shm-usage")
```

Comment:
Just for a future refactor, what do you think if we move these configuration flags into the Configuration object?